test/Serialization/Matrices.jl

```diff
@@ -4,14 +4,14 @@ K, a = number_field(q)
 Z7 = residue_ring(ZZ, 7)[1]
 Z7t, t = polynomial_ring(GF(7), "t")
 Fin, d = finite_field(t^2 + t + 3)
-Frac = fraction_field(R)
+FracFieldElem = fraction_field(R)
```

Comment:
this and the change lower in the file are potentially dangerous. please change them to a variable name that's not used by type already